src/main/flight/pid.c

```diff
@@ -812,13 +815,56 @@ static float getSterm(int axis, const pidProfile_t *pidProfile)
 #endif
 }
 
+NOINLINE static void calculateSpaValues(const pidProfile_t *pidProfile)
+{
+#ifdef USE_WING
+    for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
+        float currentRate = getSetpointRate(axis);
+        pidRuntime.spa[axis] = smoothStepDownTransition(
+            fabs(currentRate), pidProfile->spa_center[axis], pidProfile->spa_width[axis]);
```

Comment:
Waind pulling in `doble` math
```suggestion
            fabsf(currentRate), pidProfile->spa_center[axis], pidProfile->spa_width[axis]);
```

Reply:
fixed